packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -154,57 +151,53 @@ class ProgressivePaginator extends React.Component {
     return (
       <div className={cx(['paginator'])} role="navigation" aria-label="pagination">
         <div>
-          <button
-            aria-disabled={selectedPage === 1}
-            aria-label={intl.formatMessage({ id: 'Terra.paginator.first' })}
+          <PaginatorButton
+            ariaDisabled={selectedPage === 1}
+            ariaLabel={intl.formatMessage({ id: 'Terra.paginator.first' })}
             className={cx(['nav-link', selectedPage === 1 ? 'is-disabled' : null])}
             tabIndex={selectedPage === 1 ? null : '0'}
             onClick={this.handlePageChange(1)}
             onKeyDown={this.handleOnKeyDown(1)}
-            type="button"
           >
             {intl.formatMessage({ id: 'Terra.paginator.first' })}
-          </button>
-          <button
-            aria-disabled={selectedPage === 1}
-            aria-label={intl.formatMessage({ id: 'Terra.paginator.previous' })}
+          </PaginatorButton>
+          <PaginatorButton
+            ariaDisabled={selectedPage === 1}
+            ariaLabel={intl.formatMessage({ id: 'Terra.paginator.previous' })}
             className={cx(['nav-link', 'previous', 'icon-only', selectedPage === 1 ? 'is-disabled' : null])}
             tabIndex={selectedPage === 1 ? null : '0'}
             onClick={this.handlePageChange(previousPageIndex)}
             onKeyDown={this.handleOnKeyDown(previousPageIndex)}
```

Comment:
It doesn't look like these `onKeyDown` props are being used and executed by the `PaginatorButton`. Am I missing it?

Reply:
Looks like this was leftover from moving things around. Removed! [3c4f04a](https://github.com/cerner/terra-core/pull/2686/commits/3c4f04a82bd80d77dc4fc4f35ba36c5d85deb003)